contracts/SubdomainRegistrar.sol

```diff
@@ -282,6 +303,37 @@ contract SubdomainRegistrar is RegistrarInterface {
         DomainUpgraded(label, name);
     }
 
+    function stop(address _migration) not_stopped registrar_owner_only {
```

Comment:
I think we need a way to stop and to start the migration separately; we may detect a problem before we have a solution.

Reply:
@Arachnid good point